CoffeeChat/Onboarding/GoalsViewController.swift

```diff
@@ -89,9 +89,9 @@ class GoalsViewController: UIViewController {
         setupConstraints()
     }
     
-    override func viewWillAppear(_ animated: Bool) {
+    override func viewDidAppear(_ animated: Bool) {
         getUserGoals()
-        super.viewWillAppear(animated)
+        super.viewDidAppear(animated)
```

Comment:
I would put this first. It feels strange to put it on the end but that may just be me.